openhtf/io/frontend/templates/test.html

```diff
@@ -12,6 +12,17 @@
 
 </div>
 {% else %}
+  <h2>{{state['test']['record']['dut_id']}}</h2>
+  {% if state['test']['status'] == 'PASS' %}
+    <h1 style="color:green;">
+  {% elif state['test']['status'] == 'FAIL' %}
+    <h1 style="color:red;">
+  {% else %}
+    <h1 style="color:orange;">
+  {% end %}
+  {{state['test']['status']}}</h1>
+
+{% if state['framework']['status'] == 'EXECUTING' %}
```

Comment:
in the templates, it should be possible to do state.framework.status and it'll do the right thing automatically, no?